### PR TITLE
switch for tetgen-options added

### PR DESCRIPTION
**Pull request by BalticPinguin**
Hi,
I took Roy's suggestion now for implementing the additional switch.
I hope, this meets your expectations and I didn't make any mistake in this part.
Best,
Hubert

### CODE REVIEW

File: include/mesh/mesh_tetgen_interface.h [hunk 1]
```diff
@@ -155,6 +159,12 @@ class TetGenMeshInterface
   std::vector<unsigned> _sequential_to_libmesh_node_map;
 
   /**
+   * Parameter controling the behaviour of tetgen.
+   * By default quiet.
+   */
+  std::string _switches = "Q";
```
Comment (jwpeterson) [inline include/mesh/mesh_tetgen_interface.h]: I believe this (inline initialization of non-const class members) is a C++11 only feature, although it was not caught by our "no C++11" test for some reason.

Comment (jwpeterson) [inline include/mesh/mesh_tetgen_interface.h]: Oh, probably because we don't test with `--disable-strict-lgpl`, so this new code is not even being tested.

Comment (jwpeterson) [inline include/mesh/mesh_tetgen_interface.h]: I'll add a quick fix for this, otherwise it looks fine.

Comment (BalticPinguin) [inline include/mesh/mesh_tetgen_interface.h]: Ok; I changed this line now.

File: src/mesh/mesh_tetgen_interface.C [hunk 2]
```diff
@@ -39,8 +39,22 @@ TetGenMeshInterface::TetGenMeshInterface (UnstructuredMesh & mesh) :
   _mesh(mesh),
   _serializer(_mesh)
 {
+  _switches="Q";
```
Comment (jwpeterson) [inline src/mesh/mesh_tetgen_interface.C]: Thanks for fixing this, but it should just be initialized in the initialization list (two lines up) like the other members.

**Comment by jwpeterson** [comment]
:+1: thanks!

**Pull request merged**